libraries/common/option.effekt

```diff
@@ -49,6 +49,15 @@ def option[A, E](proxy: on[E]) { p: => A / Exception[E] }: Option[A] =
   try { Some(p()) }
   with Exception[E] { def raise(_, _) = None() }
 
+/// Applies a function `f` to two `Option` values:
+/// returning `None` if either of the arguments is `None`,
+/// otherwise returning `Some` of the result.
```

Comment:
```suggestion
/**
 * Applies a function `f` to two `Option` values:
 * returning `None` if either of the arguments is `None`,
 * otherwise returning `Some` of the result.
 **/
```

Reply:
This one is my fault since it's from my `suggestion`: if we're going to phase out block comments, it might be better to go full "`///` means doc-comment". But yes, it's technically not consistent with the rest of the style.
I'd have also expected some nitpicking: is this `zip`, `map2`, `zipWith`...?